concordium-sdk/src/main/java/com/concordium/sdk/responses/getancestors/BlockAncestors.java

```diff
@@ -0,0 +1,43 @@
+package com.concordium.sdk.responses.getancestors;
+
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.transactions.Hash;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.common.collect.ImmutableList;
+import concordium.ConcordiumP2PRpc;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.val;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+@Builder
+@Getter
+/**
+ * Represents Ancestors of a Block.
+ */
+public class BlockAncestors {
```

Comment:
I don't think this abstraction provides much functionality, so I propose to simply delete it and return `ImmutableList<Hash>`.